frontend/src/app/main/ui/workspace/sidebar/history.cljs

```diff
@@ -301,8 +302,10 @@
       (when (:detail entry)
         [:div {:class (stl/css-case :history-entry-summary-button true
                                     :button-opened @show-detail?)
-               :on-click #(when (:detail entry)
-                            (swap! show-detail? not))}
+               :on-click (fn [event]
```

Comment:
Consider also:
- adding use-fn for this callback (probably using data-detail attribute mechanism for passing state)
- adding `{::mf/props :obj}` on component (taking advantage of the fact that you touch the component, you add a performance improvement that is very easy to add)

Reply:
Fixed